<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="firstImage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>761</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="setColorDistanceSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>22</width>
      <height>491</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="manhattanRadioBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Manhattan</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="euclideanRadioBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Euclidean</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="chebyshevRadioBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Chebyshev</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="minkowskiRadioBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Minkowski</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="chooseFileBtn"/>
   <addaction name="separator"/>
   <addaction name="chooseColorBtn"/>
   <addaction name="separator"/>
   <addaction name="processImageBtn"/>
   <addaction name="separator"/>
   <addaction name="saveImageBtn"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="chooseFileBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/dir.png</normaloff>:/images/icons/dir.png</iconset>
   </property>
   <property name="text">
    <string>Choose file</string>
   </property>
   <property name="toolTip">
    <string>Choose file</string>
   </property>
  </action>
  <action name="chooseColorBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/colors.png</normaloff>:/images/icons/colors.png</iconset>
   </property>
   <property name="text">
    <string>Choose color</string>
   </property>
   <property name="toolTip">
    <string>Choose color</string>
   </property>
  </action>
  <action name="processImageBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/process.png</normaloff>:/images/icons/process.png</iconset>
   </property>
   <property name="text">
    <string>Process image</string>
   </property>
   <property name="toolTip">
    <string>Process image</string>
   </property>
  </action>
  <action name="saveImageBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/save.png</normaloff>:/images/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save image</string>
   </property>
   <property name="toolTip">
    <string>Save image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>setColorDistanceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>665</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
